src/com/videojs/providers/HTTPVideoProvider.as

```diff
@@ -301,7 +301,6 @@ package com.videojs.providers{
                 // if the asset is already loading
                 if (_hasEnded) {
                   _hasEnded = false;
-                  _ns.seek(0);
```

Comment:
This is what makes it that if you seek immediately after ending, you actually start at 0 instead of where you seeked?
Though, changing this seems like it might break replay functionality when you hit play after the video has ended.